listeners.py

```diff
@@ -121,6 +178,8 @@ def save_artifacts(self):
                             ignore=shutil.ignore_patterns(
                                 '*.socket-iproto', '*.socket-admin',
                                 '*.sock', '*.control'))
+        shutil.copytree(os.path.join(vardir, 'statistics'),
+                        os.path.join(artifacts_dir, 'statistics'))
```

Comment:
NB: I guess you'll skip this directory creation, when there is no /proc (on Mac OS), so we should not forget to skip copying here in the case.

Reply:
However statistics will be here always after the patch with adding timings. I would add `if os.path.isdir()` anyway.

Reply:
Ah, I see, the directory is created always. Okay, at least it simplifies the code.